kmm/positions.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+import pandas as pd
+from pydantic import validate_arguments
+
+import kmm
+
+
+class Positions(kmm.FunctionalBase):
+
+    dataframe: pd.DataFrame
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    @staticmethod
+    @validate_arguments
+    def from_path(path: Path):
+        """
+        Loads positions from .kmm or .kmm2 file.
+        """
+        return Positions(dataframe=kmm.read_raw(path))
+
+    @staticmethod
+    @validate_arguments
+    def read_sync_adjust(
+        kmm_path: Path,
+        header_path: Path,
+        adjustment: int = kmm.ADJUST_WIRE_CAMERA,
```

Comment:
Type hint with enum

Reply:
> What is the terminology that you decided on?
> `positions` is the wrapper class?
> 
> What do you think about having the functional interface in folders like:
> 
> ```
> kmm/
>   positions/
>       positions.py
>       read.py
>       sync.py
>       adjust.py
>   header/
>       header.py
>       read.py
>       ...
> ```

I think this sounds good but the interface becomes a little strange then since we would like to expose `Positions` directly in kmm, so the root `__init__.py` would be like this
```
...
from kmm import positions
from kmm import header
from kmm.positions.positions import Positions
from kmm.header.header import Positions
```
or maybe that's fine?

Reply:
I like that pattern. It's also what we do in .jsx but we call the main file "index.jsx"